cupy/cuda/nccl.pxd

```diff
@@ -14,3 +14,18 @@ cpdef enum:
     NCCL_DOUBLE = 4
     NCCL_INT64 = 5
     NCCL_UINT64 = 6
+
+    NCCL2_INT8 = 0
```

Comment:
Thanks for the pointer, @shu65 !

I quickly went through it and now I understand that it is certainly not necessary to modify the chainer with your approach. All right, I will update that way.